### PR TITLE
chore: 🤖 Fix `getPermissions` tests for 100% coverage

### PR DESCRIPTION
**Pull request by prashantasdeveloper**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -93,7 +93,8 @@
     "webpack": "^5.69.1",
     "webpack-cli": "^4.9.2",
     "webpack-dev-server": "^4.7.4",
-    "websocket": "^1.0.31"
+    "websocket": "^1.0.31",
```
Comment (polymath-eric) [inline package.json:96]: I don't think we need `websocket` here since its in normal dependencies.

**Review by polymath-eric — COMMENTED** [review]
LGTM, one nitty comment.

**Review by polymath-eric — APPROVED** [review]

**Review by monitz87 — APPROVED** [review]

**Pull request merged**